ProductionModePlugin.js

```diff
@@ -65,6 +73,7 @@ ProductionModePlugin.prototype.apply = function(compiler) {
     var request = this.state.current.request;
     if(param.isString() && param.string === "globalize" &&
           !util.isGlobalizeModule(request) &&
+          !isTransformExcluded(request) &&
           !(globalizeCompilerHelper.isCompiledDataModule(request))) {
```

Comment:
For clarity, I am wondering if we can leverage your change to clean up this former mess.